ooniprobe/Model/Database/Measurement.mm

```diff
@@ -96,4 +96,41 @@ -(void)deleteObject{
     [self remove];
 }
 
+-(void)getExplorerUrl:(void (^)(NSString*))successcb onError:(void (^)(NSError*))errorcb {
+    NSURLComponents *components = [[NSURLComponents alloc] init];
+    components.scheme = @"https";
+    components.host = @"api.ooni.io";
+    components.path = @"/api/v1/measurements";
+    if ([self.test_name isEqualToString:@"web_connectivity"])
+        components.query = [NSString stringWithFormat:@"report_id=%@&input=%@",
+                            self.report_id, self.url_id.url];
+    else
+        components.query = [NSString stringWithFormat:@"report_id=%@",
+                        self.report_id];
+    NSURL *url = components.URL;
+    NSURLSessionDataTask *downloadTask = [[NSURLSession sharedSession]
+     dataTaskWithURL:url
```

Comment:
Can you indent the continuation code using two spaces rather than one space? (Or four spaces if you fancy that more, I'm mostly concerned that I see a one-space-indentation unusual)

Reply:
I can add random spaces but this is how the XCode auto-indentation indented this line

Reply:
That surprises me, however let's keep nonetheless it like XCode indents it by default.